scripts/publish/src/slackNotification.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-console */
+const request = require('request');
+const chalk = require('chalk');
+
+// This value will need to be made available to the script in jenkins.
+const slackToken = 'XXXXXXXXX/XXXXXXXXX/XXXXXXXXXXXXXXXXXXXXXXXX';
```

Comment:
Can be passed in as a `process.env.SLACK_TOKEN` & we ensure Jenkins has this environment variable.